src/core_plugins/kibana/public/dashboard/panel/dashboard_panel_container.js

```diff
@@ -0,0 +1,50 @@
+import { connect } from 'react-redux';
+
+import { DashboardPanel } from './dashboard_panel';
+import { DashboardViewMode } from '../dashboard_view_mode';
+
+import {
+  renderEmbeddable,
+  maximizePanel,
+  minimizePanel,
+  deletePanel,
+  destroyEmbeddable
+} from '../actions';
+
+import {
+  getEmbeddable,
+  getFullScreenMode,
+  getViewMode,
+  getEmbeddableTitle,
+  getEmbeddableEditUrl,
+  getMaximizedPanelId,
+  getEmbeddableError,
+} from '../reducers';
+
+const mapStateToProps = ({ dashboardState }, { panelId }) => {
```

Comment:
Yea, I worry if I put all selectors at the root of the core kibana store it would become a huge file over time (especially because it would include state for all the core apps, Visualize, Discover, Dashboard, .... etc).  I'm also not just sure how I feel about all that knowledge about what exists below the state tree in the upper most container.  

I could pass state and then change `kibana/public/dashboard/reducers/index.js` to wrap all the state passed into all those selectors in `getDashboard` from `kibana/public/reducers.js`, but then I'd be breaking the contract I currently abide by where each reducer gets state at it's own level.

I'm not really sure what the best approach is here.  I think I'll leave as is for now and wait for more discussions before converting.